source/Hex.lpi

```diff
@@ -278,12 +278,12 @@
     <Version Value="11"/>
     <PathDelim Value="\"/>
     <Target>
-      <Filename Value="../bin/$(TargetCPU)-$(TargetOS)/Hex"/>
+      <Filename Value="..\bin\Hex"/>
```

Comment:
As above...

Reply:
It wasn't me who changed this, it was Lazarus (Fedora). Sorry.